processing-service-core/api/models/db/globalConfigStore.js

```diff
@@ -192,11 +210,21 @@ function setGlobalConfig(key, defaultValue, description) {
  */
 function setGlobalConfigWithScope(key, defaultValue, description, scope) {
     var deferredSet = Q.defer();
+    var processedScope;
+    switch(scope){
+        case "REPOSITORY":
+            processedScope = 1;
+            logger.debug("Scope set to 1");
+            break;
+        default:
```

Comment:
This should be `case "TENANT":` and the default branch should throw an error if it doesn't recognize the scope.